### PR TITLE
Reorganize dmpr into multiple files

### PR DESCRIPTION
**Pull request by reisub-de**
looks better, shorter scroll paths ^^

### CODE REVIEW

File: dmpr/path.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+from .exceptions import InternalException
+
+
+def dict_reverse_lookup(d: dict, value):
+    return list(d.keys())[list(d.values()).index(value)]
+
+
+class Path:
+    """Auxiliary class to handle paths, appending and their link
```
Comment (hgn) [inline dmpr/path.py]: I prefer:

- new style classes (`class Foo(object)`)
- little different comment formatting

```
class Path(object):
    """
    Lorum ipsum shizle dizzle
    """

    def __init__(self):
        """
        Foo Bar Comment
        """
        pass
```

**Review by hgn — CHANGES_REQUESTED** [review]

File: dmpr/message.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        if self.node_data != msg_node_data:
+            update = True
+            self.node_data = msg_node_data
+
+        self._base = BaseMessage()
+        self._base.apply_base_data(self)
+
+        return update
+
+    def _apply_partial(self, msg: dict) -> bool:
```
Comment (hgn) [inline dmpr/message.py]: sometimes we need more comments. The code is clear but it is still not *that* easy to know what the functions does and what particular loops do. Some small comments before functional blocks are helpful too, beside a general comment at the top of the function what a function in general do. Right?

File: dmpr/policies.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    """ Route via the highest-bandwidth path"""
+    name = 'highest-bandwidth'
+
+    @staticmethod
+    def _acc_bw(path: Path) -> int:
+        return min(path.attributes[link]['bandwidth'] for link in path.links)
+
+    @AbstractPolicy.with_path_cache
+    def path_cmp_key(self, path: Path):
+        # the minimum bandwidth of the path while slightly preferring
```
Comment (hgn) [inline dmpr/policies.py:69]: like this comment, really helpful

**Review by hgn — CHANGES_REQUESTED** [review]

File: dmpr/path.py (hunk 1 above)
Comment (reisub-de) [reply dmpr/path.py]: In python 3 all classes are new style classes, but I can change that to the explicit form.

File: dmpr/message.py (hunk 2 above)
Comment (reisub-de) [reply dmpr/message.py]: Yeah, comments need to be improved, I will work on that.

**Comment by reisub-de** [comment]
Added and improved comments

**Pull request merged**